deploy/operator.yaml

```diff
@@ -223,3 +191,14 @@ webhooks:
         apiGroups: ["*"]
         apiVersions: ["v1alpha1"]
         resources: ["nodenetworkconfigurationpolicies", "nodenetworkconfigurationpolicies/status"]
+---
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  namespace: {{ .HandlerNamespace }}
+  name: {{template "handlerPrefix" .}}nmstate-webhook
+spec:
+  minAvailable: "90%"
```

Comment:
We have two replicas, 90% would mean we need 2/2. I think we should set it so it requires at least one instance to be up and running.